src/lib/shim/src/reinit_auxvec_random.rs

```diff
@@ -1,3 +1,5 @@
+use core::{ffi::c_ulong, mem::size_of, ptr};
```

Comment:
`size_of` is part of the core prelude, so you shouldn't need to include it here. (I'm surprised this doesn't give a clippy warning.)